.scripts/electron-desktop-environment/desktop-environment-manager.ts

```
@@ -19,74 +19,128 @@
-			this.instance.fileDir,
-			this.instance.fileName.concat(`.ts`)
-		);
-		if (fs.existsSync(filePath)) {
-			fs.unlinkSync(filePath);
-			fs.writeFileSync(
-				filePath,
-				this.instance.content(environment, this.instance.isProd)
-			);
-			console.log(`✔ environment ${filePath} updated.`);
-			return;
+		const environment: IDesktopEnvironment = { ...this.environment };
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)